### DISCUSSION TITLE
Run script after/before suspending/hibernating and when system starts/shuts down

### DISCUSSION THREAD

**Discussion by realroot2185** [davmac314/dinit#349] [Q&A] [answered]
Is there a folder to place scripts that will be executed in those moments?

For example I found out about `/etc/local.d `  here https://forum.artixlinux.org/index.php/topic,7059.0/topicseen.html.
I do not have the folder.

I know that elogind can do something at least for suspending/hibernating.
Can you use dinit instead?

**Comment by davmac314** [comment, marked as answer]
> Is there a folder to place scripts that will be executed in those moments?
> I know that elogind can do something at least for suspending/hibernating.
Can you use dinit instead?

No, dinit has no direct support for any kind of power management other than for shutdown. This functionality can be provided by other packages.

For what it's worth the Dinit documentation is quite complete. If something isn't mentioned in the documentation then it's not a feature.

> For example I found out about /etc/local.d here

That's nothing to do with dinit itself. Any functionality for executing things from `/etc/local.d` is provided by the distribution (possibly they implement it as a dinit service, but I don't have anything to do with it).

**Reply by realroot2185** [reply]
What other packages can do that? For example I am running seatd instead of elogind in one computer.

**Reply by davmac314** [reply]
I think pm-utils runs scripts on suspend/hibernate/resume. You would have to check its documentation.
